### DISCUSSION TITLE
单机版的milvus2.0数据是保存在什么地方，怎么作迁移

### DISCUSSION THREAD

**Discussion by lanceMsf** [milvus-io/milvus#17666] [Q&A and General discussion]
单机版的milvus2.0数据是保存在什么地方，怎么作迁移

**Comment by xiaofan-luan** [comment]
通过milvus.yaml
迁移的话比较复杂，要考虑etcd，standalone和本地磁盘上的文件，后面我们会出工具

**Reply by SilentEchoe** [reply]
现在有工具能够实现milvus的数据迁移吗？

**Reply by yhmo** [reply]
> 现在有工具能够实现milvus的数据迁移吗？

https://github.com/zilliztech/milvus-backup
这时目前推荐的工具。该工具的原理是把一张表的数据文件拷贝到S3/MinIO的另一个目录里，然后再导入另一个milvus server。

**Comment by SilentEchoe** [comment]
十分谢谢您的回复，这对我帮助很大


在 2024年1月15日 ***@***.***> 写道：


现在有工具能够实现milvus的数据迁移吗？
https://github.com/zilliztech/milvus-backup
 这时目前推荐的工具。该工具的原理是把一张表的数据文件拷贝到S3/MinIO的另一个目录里，然后再导入另一个milvus server。
